recipes/json-schema-validator/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+from conans.tools import Version
+from conans.errors import ConanInvalidConfiguration, ConanException
+
+class JsonSchemaValidatorConan(ConanFile):
+    name = "json-schema-validator"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/pboettch/json-schema-validator"
+    description = "JSON schema validator for JSON for Modern C++ "
+    topics = ("json-schema-validator", "modern-json",
+              "schema-validation", "json")
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt"]
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    short_paths = True
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        compiler_version = Version(self.settings.compiler.version)
+        if self.settings.os == "Windows"and self.settings.compiler == "Visual Studio" and compiler_version < "16":
+            try:
+                # Force check to ensure compatability
+                tools.check_min_cppstd(self, "17") # There's an issue in upstream forcing a higher cppstd version. See https://github.com/pboettch/json-schema-validator/issues/106
+            except ConanInvalidConfiguration:
+                raise
+            except ConanException:
+                self.output.warn("Unnable to determine the default standard version of the compiler")
+        elif self.settings.compiler == "clang" and compiler_version < "4":
+            try:
+                # Force check to ensure compatability
+                tools.check_min_cppstd(self, "11") # Clang 3.9 requires this to be set to ignore the issue above https://releases.llvm.org/3.9.1/tools/clang/docs/ReleaseNotes.html#c-1z-feature-support
+            except ConanInvalidConfiguration:
+                raise
+            except ConanException:
+                self.output.warn("Unnable to determine the default standard version of the compiler")
```

Comment:
Do not raise an exception. You can't block because you didn't test other compilers.